src/common/gateway/gateway.service.ts

```diff
@@ -19,9 +19,11 @@ import { BinaryUtils } from "@multiversx/sdk-nestjs-common";
 import { ApiService, ApiSettings } from "@multiversx/sdk-nestjs-http";
 import { GuardianResult } from "./entities/guardian.result";
 import { TransactionProcessStatus } from "./entities/transaction.process.status";
+import { TxPoolGatewayResponse } from "./entities/transaction.pool";
 
 @Injectable()
 export class GatewayService {
+  private snapshotlessRequestsSet: Set<String> | undefined;
```

Comment:
you can initialize this directly here. we can then make the field read-only and skip the undefined marker

Reply:
done